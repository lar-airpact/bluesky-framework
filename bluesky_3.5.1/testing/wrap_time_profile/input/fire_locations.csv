date_time,event_id,event_name,id,type,latitude,longitude,area
20110405,SF11E1600994,Flint Hills,SF11C1736234-6432886710,RX,36.676,-96.464,3191.852657
