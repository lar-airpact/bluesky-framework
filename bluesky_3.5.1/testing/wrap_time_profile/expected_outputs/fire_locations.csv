id,event_id,latitude,longitude,type,area,date_time,slope,state,country,fips,scc,fuel_1hr,fuel_10hr,fuel_100hr,fuel_1khr,fuel_10khr,fuel_gt10khr,shrub,grass,rot,duff,litter,moisture_1hr,moisture_10hr,moisture_100hr,moisture_1khr,moisture_live,moisture_duff,consumption_flaming,consumption_smoldering,consumption_residual,consumption_duff,min_wind,max_wind,min_wind_aloft,max_wind_aloft,min_humid,max_humid,min_temp,max_temp,min_temp_hour,max_temp_hour,sunrise_hour,sunset_hour,snow_month,rain_days,VEG,canopy,event_name,fccs_number
SF11C1736234-6432886710,SF11E1600994,36.676,-96.464,RX,3191.852657,201104050000-06:00,10.0,OK,USA,40113,2810015000,0.4,0.2,0.0,0.0,0.0,0.0,0.27,2.1,0.0,0.0,0.9,10.0,12.0,12.0,22.0,130.0,150.0,3.64986483062,0.214515194734,0.0,0.0,6.0,6.0,6.0,6.0,40.0,80.0,13.0,30.0,4,14,6,18,5,8,Tall fescue - Foxtail - Purple bluestem grassland,0.54,Flint Hills,133
